ns1/resource_datasource.go

```diff
@@ -55,6 +58,13 @@ func DataSourceRead(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*ns1.Client)
 	s, _, err := client.DataSources.Get(d.Id())
 	if err != nil {
+		// No custom error type is currently defined in the SDK for a data feed.
+		if strings.Contains(err.Error(), "source not found") {
+			log.Printf("[DEBUG] NS1 data feed (%s) not found", d.Id())
```

Comment:
Possible typo in message and comment?  This is working with `client.DataSources` so shouldn't this be `[DEBUG] NS1 data source (%s) not found`